src/components/Vault/VaultDepositForm.tsx

```diff
@@ -132,7 +119,12 @@ const VaultDepositForm: FC<VaultProps> = (props) => {
     ...vaultContract,
     functionName: "depositUnderlying",
     enabled: enableDepositUnderlying,
-    args: [inputTokenAddress, userAddress ?? "0x", value, minAmount],
+    args: [
+      inputTokenAddress,
+      userAddress ?? "0x",
+      value,
+      BigNumber.from(previewDeposit.data?.resultWei ?? 0),
```

Comment:
I think if the preview BE methods don't take into account the slippage as we said, the preview.data.resultWei is still wrong. Right?

Reply:
For AMMCompounders, the BE returns with slippage accounted for. For TokenCompounders, the BE does NOT return with slippage accounted for. I let dybbuk know -- this should be resolved on the BE. 

All preview endpoints are broken currently, but this should work once they are fixed + slippage is handled on tokens.